<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2549</width>
    <height>1517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2549</width>
    <height>1517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:#89a3e3;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>601</width>
      <height>1331</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>4</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
color:#2f3640;
}
QTabWidget::pane{
border-radius:30px;

background-color:#00000000;
}
QWidget{
   background-color: #ecf0f1;
	border-radius:20px;
}
QTabBar{
background-color: #00000000;
}
QTabBar::tab {
	font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
    background-color: #2f3640;
    color: #fff;

	height:80px;
	width:160px;
    border-radius: 20px;
    border-top-right-radius: 50px;  
border-top-left-radius: 50px;  

}
QTabBar::tab:top:selected {
    color: #2f3640;
	background-color:#ecf0f1;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="inputTab">
     <property name="styleSheet">
      <string notr="true">background-color:#00000000;</string>
     </property>
     <attribute name="title">
      <string>입력</string>
     </attribute>
     <widget class="QListWidget" name="fileList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>501</width>
        <height>511</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
color:#2f3640;
}

QListWidget::item::hover{
	background-color:#f7ede2;

}
QListWidget::item::clicked{
	color:#000000;


}
QListWidget::item{
	background-color:#f0c6bf;
	border-radius: 50px;
	height:80px;
	padding:10px;

}
QListView::icon {
	left:40px;
}
QListView::text {
    left: 70px;
}</string>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::LinkAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="batchSize">
       <number>100</number>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="inputList">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>760</y>
        <width>491</width>
        <height>451</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
color:#2f3640;
}

QListWidget::item::hover{
	background-color:#f7ede2;

}
QListWidget::item::clicked{
	color:#000000;


}
QListWidget::item{
	background-color:#f0c6bf;
	border-radius: 50px;
	height:80px;
	padding:10px;

}
QListView::icon {
	left:40px;
}
QListView::text {
    left: 70px;
}</string>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::CopyAction</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;NanumSquare&quot;;
</string>
      </property>
      <property name="text">
       <string>파일</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>700</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;NanumSquare&quot;;
</string>
      </property>
      <property name="text">
       <string>실시간 입력</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>646</y>
        <width>530</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e3e3e3</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modelTab">
     <property name="styleSheet">
      <string notr="true">background-color:#00000000;</string>
     </property>
     <attribute name="title">
      <string>모델</string>
     </attribute>
     <widget class="QListWidget" name="modelList">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>70</y>
        <width>491</width>
        <height>1151</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>3</number>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{
	font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
	color:#2f3640;
}

QListWidget::item::hover{
	background-color:#cce3de;

}
QListWidget::item::clicked{
	color:#000000;


}
QListWidget::item{
	background-color:#a4c3b2;
	border-radius: 50px;
	height:80px;
	padding:10px;

}
QListView::icon {
	left:40px;
}
QListView::text {
    left: 70px;
}</string>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="mainList">
    <property name="geometry">
     <rect>
      <x>826</x>
      <y>160</y>
      <width>561</width>
      <height>1131</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	font: 18pt &quot;나눔스퀘어_ac Bold&quot;;
border:5px;
	background-color:  #ecf0f1;
border-radius: 20px;

}

QListWidget::item::hover{
	background-color:#f5ebe0;

}
QListWidget::item{
	background-color:#e3d5ca;
	border-radius: 10px;
	height:90px;



}

QListView::icon {
	width:0px;
}</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::Batched</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>1310</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 14pt &quot;NanumSquare&quot;;


</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>1310</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 14pt &quot;NanumSquare&quot;;


</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>160</y>
      <width>961</width>
      <height>1251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color:#ecf0f1;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>471</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="plyOut"/>
    </widget>
    <widget class="QTextBrowser" name="outputText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>851</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;NanumSquare&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
    background-color: #2f3640;
    color: #fff;

	height:80px;
	width:150px;
    border-radius: 20px;
    border-top-right-radius: 50px;  
border-top-left-radius: 50px;  </string>
    </property>
    <property name="text">
     <string>보드</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
    background-color: #2f3640;
    color: #fff;

	height:80px;
	width:150px;
    border-radius: 20px;
    border-top-right-radius: 50px;  
border-top-left-radius: 50px;  </string>
    </property>
    <property name="text">
     <string>출력</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>3040</x>
      <y>920</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;나눔스퀘어_ac Bold&quot;;
    background-color: #2f3640;
    color: #fff;

	height:80px;
	width:150px;
    border-radius: 20px;
    border-top-right-radius: 50px;  
border-top-left-radius: 50px;  </string>
    </property>
    <property name="text">
     <string>모델 보드</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
